Chiazokam/Politico
# #163351835 create the get all parties endpoint

#### What does this PR do?
This PR tests and creates the GET all parties endpoint

#### Description of Task to be completed?
- write tests for all the functions
- create the endpoint
- validate the existence of the party in create party

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-datastructures-get-all-parties-163351835`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Make a `GET` request to `http://localhost:3000/api/v1/parties `
- You should receive a successful response

#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
[#163351835](https://www.pivotaltracker.com/story/show/163351835)

####  Images

![image](https://user-images.githubusercontent.com/26940294/51778633-1dcb6c00-2103-11e9-9895-c587211d9fa3.png)
